config/environments/production.rb

```diff
@@ -91,4 +91,9 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+
+  config.action_cable.mount_path = '/cable'
+  config.action_cable.url = 'wss://localhost:3000/cable'
+  config.action_cable.allowed_request_origins = [ /http:\/\/localhost:*/ ]
```

Comment:
Not sure that it will work on production

Reply:
To not forget this settings, I will change them when deploy